### PR TITLE
added new project

### PR DESCRIPTION
**Pull request by slayerrr12**
## Related Issue

https://github.com/priyankarpal/ProjectsHut/issues/801

## Description

addition of new project

## Screenshots

image

### CODE REVIEW

**Comment by slayerrr12** [comment]
@priyankarpal pls review this pr

File: .env.example
Comment (priyankarpal) [inline .env.example:1]: add this file

File: src/DB/projects.json [hunk 1]
```diff
@@ -919,6 +919,12 @@
         "title": "BotForge-Discord",
         "description": "A discord bot that does wonders ",
         "tech": ["DiscordJS", "Node.js"]
+      },
+      {
+        "link": "https://github.com/slayerrr12/Cloned-APPS",
+        "title": "Cloned-APPS",
+        "description": "clones i have created for learning purpose in my web dev journey",
+        "tech": ["HTML", "CSS", "Node.js", "VANILLA JAVASCRIPT"]
```
Comment (priyankarpal) [inline src/DB/projects.json:927]: only Javascript is ok 
``` suggestion
        "tech": ["HTML", "CSS", "Node.js", "Javascript"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: .env.example
Comment (slayerrr12) [reply .env.example:1]: ok

**Pull request closed without merging**